src/main/scala/circt/Intrinisic.scala

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/** This has to be in a separate file and package to generate the correct class
+  *  in the annotation.  This is supper annoying.
+  */
```

Comment:
Unnecessary comment and sort of weird. Yes, in Scala you set the package on a per file basis.

Reply:
No, it's that I can't control the class string in the json annotation.  It's weird that class structure implementation details are reflected in the output.

Reply:
Specifically, this needs to line up with "circt.Intrinsic` in the JSON class name so that it works here: https://github.com/llvm/circt/blob/cd0cae2fd1b0c765b90bc51f87025bb11196c6c4/lib/Dialect/FIRRTL/Transforms/LowerAnnotations.cpp#L317

Reply:
Why not change the name in CIRCT to match the one in chisel3? Seems like an unnecessary restriction if we have to stick to the one currently used in CIRCT.

Reply:
```suggestion
/** This has to have the class name `circt.Intrinsic` to match the class name 
  * for the annotation that CIRCT is expecting.
  */
```

Reply:
in other words, can we file a bug / feature request for the "annoying" part

Reply:
> Why not change the name in CIRCT to match the one in chisel3? Seems like an unnecessary restriction if we have to stick to the one currently used in CIRCT.

We did once already, in response to a capitalization comment above.  But the annoying part is that annotation text is tied to Scala class name.